src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/custom.py

```diff
@@ -228,7 +228,7 @@ def db_create(
 
     # Verify edition
     edition = kwargs.get('edition')  # kwags['edition'] throws KeyError if not in dictionary
-    if edition and edition.lower() == DatabaseEdition.data_warehouse.value.lower():
+    if edition and edition.lower() == DatabaseEdition.data_warehouse.value.lower():  # pylint: disable=no-member
```

Comment:
After the monitor SDK was generated using autorest 3, I actually have to update cases like this to not reference `value`.  It seems that if the SDK is generated using autorest 3, then the enum actually doesn't have a "value" field and you'll get an AttributeError (@lmazuel can you confirm?).  I'd recommend you try this out on a fresh virtual env to ensure that Pylint isn't catching something legit.

Reply:
I did this in a new virtual env, actually.